src/librustdoc/clean/types.rs

```diff
@@ -57,11 +57,18 @@ crate struct Crate {
     crate primitives: Vec<(DefId, PrimitiveType)>,
     // These are later on moved into `CACHEKEY`, leaving the map empty.
     // Only here so that they can be filtered through the rustdoc passes.
-    crate external_traits: Rc<RefCell<FxHashMap<DefId, Trait>>>,
+    crate external_traits: Rc<RefCell<FxHashMap<DefId, TraitWithExtraInfo>>>,
```

Comment:
Is `external_traits` used only for traits accessible from outside the current crate? If so, does that mean `doc(spotlight)` will no longer work for private traits? I'm assuming it will still work because of `Cache.traits`, but I want to check to be absolutely sure :)

Reply:
It doesn't impact the traits in the current crate if that's your question.